CMakeLists.txt

```diff
@@ -139,8 +139,10 @@ enable_cppcheck()
 # Pull in helper macros for working with leatherman libraries
 include(leatherman)
 
+file(GLOB_RECURSE ALL_SOURCES lib/src/*.cc lib/inc/*.hpp lib/inc/version.h exe/*.cc exe/*.hpp exe/*.h)
```

Comment:
The duplication in initializing ALL_SOURCES was to work around an earlier build issue. I thought ALL_SOURCES was being modified by one of the included cmake files. However, I checked and I was wrong. There is no need for the second initialization, here or in locales/CMakeLists.txt. I will now push the corrected commit.